com.heroicgameslauncher.hgl.yml

```
@@ -0,0 +1,158 @@
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d;vulkan/implicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+  
+  x-compat-i386-opts: &compat_i386_opts
+    prepend-pkg-config-path: /app/lib32/pkgconfig:/usr/lib/i386-linux-gnu/pkgconfig
+    ldflags: -L/app/lib32
+    append-path: /usr/lib/sdk/toolchain-i386/bin
+    env:
+      CC: i686-unknown-linux-gnu-gcc
+      CXX: i686-unknown-linux-gnu-g++
+    libdir: /app/lib32
+
+modules:
+
+  # --- Tools ---
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=/app/bin
+    cleanup: ["*"]
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: vulkan-tools
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DGLSLANG_INSTALL_DIR=/app
+      - -DVULKAN_HEADERS_INSTALL_DIR=/app
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: archive
+        url: https://github.com/KhronosGroup/Vulkan-Tools/archive/refs/tags/v1.2.186.tar.gz
+        sha256: 8ba0e21e8de0b70d31985d251d11150efee71266875262798784e9d1ec9b717b
+
+  - name: gamemode
+    buildsystem: meson
+    config-opts: &gamemode_opts
+      - -Dwith-systemd=false
+      - -Dwith-daemon=false
+      - -Dwith-examples=false
+      - -Dwith-util=false
+      - -Dwith-sd-bus-provider=no-daemon
+    sources: &gamemode_sources
+      - type: archive
+        url: https://github.com/FeralInteractive/gamemode/releases/download/1.6.1/gamemode-1.6.1.tar.xz
+        sha256: 10c2a3f142eae472f5a09e42616e38c666c05b25ca3e61e562a543bb3fda66c5
+
+  - name: gamemode-32bit
+    build-options:
+      arch:
+        x86_64: *compat_i386_opts
+    buildsystem: meson
+    config-opts: *gamemode_opts
+    sources: *gamemode_sources
+
+  - name: gamemoderun
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 data/gamemoderun -t /app/bin
+    sources: *gamemode_sources
+
+  # --- Heroic ---
+  - name: heroic
+    buildsystem: simple
+    build-commands:
+      - unappimage Heroic-*.AppImage
+      - mv squashfs-root /app/bin/heroic
+      - install -D heroic-run -t /app/bin
+    sources:
+      - type: script
+        dest-filename: heroic-run
+        commands:
+          - zypak-wrapper /app/bin/heroic/heroic "$@"
+
+      - type: file
+        x-checker-data:
+          type: electron-updater
+          url: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/latest/download/latest-linux.yml
+        url: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/download/v2.2.3/Heroic-2.2.3.AppImage
+        sha512: 952524d5f7e208eee25e5ffc53ed4b9dc240b0ffb7ea38b465114f4d65e636ecc47f0a4669e5c5f62f6744df9f9c087d07dc6690c1cc0afb6d8baaa9005df1d6
```

Nitpick: `x-checker-data` property is usually added last:
```suggestion
        url: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/download/v2.2.3/Heroic-2.2.3.AppImage
        sha512: 952524d5f7e208eee25e5ffc53ed4b9dc240b0ffb7ea38b465114f4d65e636ecc47f0a4669e5c5f62f6744df9f9c087d07dc6690c1cc0afb6d8baaa9005df1d6
        x-checker-data:
          type: electron-updater
          url: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/releases/latest/download/latest-linux.yml
```

---

fixed on latest commit